allas checks and updates (#2165)
CSCfi/csc-user-guide: docs/data/Allas/allas_encryption.md

# Tools for client side encryption for Allas

Allas is not certified as high level security storage platform and thus you should not use it to store sensitive data in readable format.

> Howerver, sensitive data can be stored to Allas if it is properly encrypted before data is transported to Allas. 

Reviewer: ```suggestion
However, sensitive data can be stored on Allas if it is properly encrypted before transfer to the object storage.
```

For sensitive data we generally recommend the [SD Connect service](../sensitive-data/sd_connect.md) that provides Web and command line interfaces 

> that automatically encrypt the data when it stored to Allas. 

Reviewer: ```suggestion
that automatically encrypt the data when it is stored to Allas. 
```

> If you for some reason don't want to use SD Connect below you can find some alternative approaches for storing your sensitive data to Allas. When you use Allas with these encryption tools, remember that:

Reviewer: ```suggestion
If you for some reason don't want to use SD Connect, below you can find some alternative approaches for storing your sensitive data to Allas. When you use Allas with these encryption tools, remember that:
```

>    1. You can store encrypted sensitive data in Allas, but are allowed to decrypt it only in environments with high enough security level. For example, in the HPC environment of CSC is NOT secure enough for sensitive data. 

Reviewer: ```suggestion
   1. You can store encrypted sensitive data in Allas, but are allowed to decrypt it only in environments with a high enough security level. For example, the CSC HPC environment (i.e. Puhti, Mahti, LUMI) is **not** secure enough for sensitive data.
```

### Asymmetric crypt4gh encryption

If you want to use asymmetric `crypt4gh` encryption, you need to have a public key file for encryption and a secret key file for decryption.

> In Puhti, you first need to make `crypt4gh` available with command:

Reviewer: ```suggestion
In Puhti, you first need to make `crypt4gh` available with the command:
```
